src/Common/TestCodeRules.ruleset

```diff
@@ -4,6 +4,8 @@
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
     <Rule Id="CA1014" Action="None" />
     <Rule Id="CA1020" Action="None" />
+    <Rule Id="CA1062" Action="None" />
+    <Rule Id="CA1303" Action="None" />
```

Comment:
Please set **Localizable(false)** on those parameters which require no localization rather then disabling the rule.